src/qt/askpassphrasedialog.cpp

```diff
@@ -23,7 +23,8 @@ AskPassphraseDialog::AskPassphraseDialog(Mode _mode, QWidget *parent) :
     ui(new Ui::AskPassphraseDialog),
     mode(_mode),
     model(0),
-    fCapsLock(false)
+    fCapsLock(false),
+    fShowPass(false)
```

Comment:
Remove this variable. Use `toggled(bool checked)` signal (http://doc.qt.io/qt-5/qabstractbutton.html#toggled) which gives the state needed.

Reply:
I had initially tried that, but toggled(bool) was not firing. It turns out I was setting the 'checkable' property incorrectly.  Fixed now.